control-plane-operator/controllers/hostedcontrolplane/clusterpolicy/config.go

```diff
@@ -67,5 +67,14 @@ func reconcileConfig(cfg *openshiftcpv1.OpenShiftControllerManagerConfig, minTLS
 			CipherSuites:  cipherSuites,
 		},
 	}
+
+	if featureGates != nil {
```

Comment:
I don't think this is needed.  Similar to `cipherSuites` input, can't we simply set the value to what was passed into this function?